domain/lib/src/usecases/myspace/get_all_document_interactor.dart

```diff
@@ -42,11 +42,16 @@ class GetAllDocumentInteractor {
 
   GetAllDocumentInteractor(this._documentRepository);
 
-  Future<Either<Failure, Success>> execute() async {
+  Future<Either<Failure, Success>> execute(Sorter sorter) async {
```

Comment:
I think you should make sorter optional and initialize it to default sorting WDYT?

Reply:
Thank u, I wrote one more function with parameters sorter. Pls pull latest code to run it again.

Reply:
I fix it at  branch https://github.com/linagora/linshare-mobile-flutter-app/pull/269